src/classes/UITL_SortContacts.cls

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
General comment - this class & the test class need to be added to the package.xml. It looks like there is a build failure on the branch that I think is related to needing to do this.

Reply:
I can't believe I completely missed that typo the whole entire time. I've updated the prefix and the places it is referenced. I've also updated the package.xml and added this new class & its test class.